dependency/actions/compile/entrypoint/extensions/extensions-8.2.json

```diff
@@ -0,0 +1,160 @@
+      "version": "1.0.18",
+      "md5": "3ca9ebc13b6b4735acae0a6a4c4f9a95"
+    },
+    "tideways_xhprof": {
+      "version": "5.0.4",
+      "md5": "68b68cd9410e62b8481445e0d89220c0"
+    },
+    "phpiredis": {
+      "version": "1.0.1",
+      "md5": "09a9bdb347c70832d3e034655b604064"
+    }
+  },
+  "pecl_extensions": [
+    {
+      "name": "ssh2",
+      "version": "1.3.1",
+      "md5": "aa29046de73f1036aea66b4a70f6f598"
+    },
+    {
+      "name": "rdkafka",
+      "version": "6.0.3",
+      "md5": "fd10faec0c599e34837a9482d3c6c801"
+    },
+    {
+      "name": "imagick",
+      "version": "3.7.0",
+      "md5": "0687774a6126467d4e5ede02171e981d"
+    },
+    {
+      "name": "gnupg",
+      "version": "1.5.1",
+      "md5": "c48f5de2f96ffebe2e18eaefff4917f9"
+    },
+    {
+      "name": "apcu",
+      "version": "5.1.22",
+      "md5": "2e1fb1f09725ada616e873c4e4012ff6"
+    },
+    {
+      "name": "igbinary",
+      "version": "3.2.14",
+      "md5": "eccbabb49090caf8c3c19dc64f16ce87"
+    },
+    {
+      "name": "LZF",
+      "version": "1.7.0",
+      "md5": "e242c66e53c0d155982713cd78b3d480"
+    },
+    {
+      "name": "mailparse",
+      "version": "3.1.4",
+      "md5": "1ab8d517fcfad5624e6fc4de71e24b47"
+    },
+    {
+      "name": "mongodb",
+      "version": "1.15.1",
+      "md5": "885483aa6a4fa7fbd723daefece974ec"
+    },
+    {
+      "name": "msgpack",
+      "version": "2.1.2",
+      "md5": "9bcaad416fc2b3c6ffd6966e0ae30313"
+    },
+    {
+      "name": "oauth",
+      "version": "2.0.7",
+      "md5": "8ea6eb5ac6de8a4ed399980848c04c0c"
+    },
+    {
+      "name": "xdebug",
+      "version": "3.2.0",
+      "md5": "2884c58679f8a41d08beb42abc156c22"
+    },
+    {
+      "name": "yaml",
+      "version": "2.2.2",
+      "md5": "22678c1238f2c6848ff7d74b780a8307"
+    },
+    {
+      "name": "phalcon",
+      "version": "5.2.1",
+      "md5": "9cd7861e2d0c6171c82e5c882a3e1ff4"
+    },
+    {
+      "name": "pdo_sqlsrv",
+      "version": "5.10.1",
+      "md5": "7db86f687b7b69326c7931a31e7a49c3"
+    },
+    {
+      "name": "stomp",
+      "version": "2.0.3",
+      "md5": "30de4089ae5c17f32617cef35dbe53e5"
+    },
+    {
+      "name": "sqlsrv",
+      "version": "5.10.1",
+      "md5": "f60a5eb24c1bba21d78cbbef05efe73d"
+    },
+    {
+      "name": "amqp",
+      "version": "1.11.0",
+      "md5": "3a0638223e320fc21f3b079198c24f12"
+    },
+    {
+      "name": "memcached",
+      "version": "3.2.0",
+      "md5": "acc58fea7b7f456408a25ac927846ad0"
+    },
+    {
+      "name": "redis",
+      "version": "5.3.7",
+      "md5": "1ed6793902214cc02467666ba69dd2be"
+    },
+    {
+      "name": "maxminddb",
+      "version": "1.11.0",
+      "md5": "5feec07108cf7898bd0591ee54b572cd"
+    },
+    {
+      "name": "psr",
+      "version": "1.2.0",
+      "md5": "07695da8376002babbce528205decf07"
+    },
+    {
+      "name": "solr",
+      "version": "2.6.0",
+      "md5": "ac5e1e1fbf28e0c543a52c533c967634"
```

Comment:
Looking at the list of extensions here, some extensions that were in the old extensions-manifests/extensions-8.2.yml file are missing, like `pdo_oci` and `oci8`, and possibly others. I didn't check 8.0 and 8.1 but we need to make sure we're not missing extensions

Reply:
@sophiewigmore I removed them because it doesn't seem that they are actually getting installed. For example, there is a check for oci8 that seems to always return false for the currently supported versions (https://github.com/cloudfoundry/binary-builder/blob/main/recipe/php_meal.rb#L202). And some extensions were causing `make test` to fail, because the test wasn't expecting to see these extensions pop up. I'll rerun the test and post the failing extensions here. If you think I should add them back, I can update the current tests to include them.

Reply:
@TisVictress that's strange... hmmm. It could be worthwhile just double checking if the versions of PHP in the buildpack.toml now have those extensions or not. If not then we're probably OK to leave them out, but I'm worried to stop supporting them if they do somehow end up available in the versions of PHP we currently ship, you know?

Reply:
Any update on this?